docs/community/contribute.rst

```diff
@@ -155,20 +155,10 @@ To understand how the below components interact with each other, refer to :ref:`
 
 To build the Flyte docs locally you will need the following prerequisites:
 
-* Install ``conda``.
-    *  We recommend Miniconda installed with an `official installer <https://docs.conda.io/projects/miniconda/en/latest/index.html#latest-miniconda-installer-links>`__.
 * Install `conda-lock <https://github.com/conda/conda-lock>`__.
-* In the ``flyteorg/flyte`` root directory run:
-    * ``conda-lock install --name monodocs-env monodocs-environment.lock.yaml``
-    * ``conda activate monodocs-env``
```

Comment:
Can we keep this so some people can still use conda to install if they don't want to run Docker locally?

Reply:
Fixed in the latest commit.